awscli/customizations/cloudtrail.py

```diff
@@ -184,6 +186,18 @@ def _call(self, options, parsed_globals):
                 'Logs will be delivered to {bucket}:{prefix}\n'.format(
                     bucket=bucket, prefix=options.s3_prefix or ''))
 
+    def _get_policy(self, key_name):
+        try:
+            data = self.s3.GetObject(
+                bucket='awscloudtrail-policy-' + self.region_name,
+                key=key_name)
+        except Exception:
+            LOG.error('Unable to get regional policy template for'
+                      ' region %s: %s', self.region_name, key_name)
+            raise
+
+        return data['Body'].read().decode('utf-8')
```

Comment:
This should probably be moved inside the try/except because we can still hit timeout/connection errors when we try to read the response body.